python/dolfinx/fem/assemble.py

```diff
@@ -344,23 +345,27 @@ def apply_lifting(
     )
     _a = [None if form is None else form._cpp_object for form in a]
     _bcs = [[bc._cpp_object for bc in bcs0] for bcs0 in bcs]
-    _cpp.fem.apply_lifting(b, _a, constants, coeffs, _bcs, x0, scale)
+    _cpp.fem.apply_lifting(b, _a, constants, coeffs, _bcs, x0, alpha)
 
 
 def set_bc(
     b: np.ndarray,
     bcs: list[DirichletBC],
     x0: typing.Optional[np.ndarray] = None,
-    scale: float = 1.0,
+    alpha: float = 1,
```

Comment:
If the function is deprecated, is there a reason for redefining the keyword making it non-backwards compatible?

Reply:
Should be changed back to `scale`.